Exercise_A/index.html

```diff
@@ -1,45 +1,270 @@
+      </div>
+    </section>
+    <aside class="bg-dark">
+      <div class="container text-center">
+        <div class="call-to-action">
+          <h2>Seriously, It's This Fucking Template !<p class="text-muted">Because nothing says hard work and talent
+              like
+              editing a few lines of text.</p>
+          </h2>
+          <a href="#" class="btn btn-default btn-lg wow tada" style="visibility: visible; animation-name: tada;">That's
+            Amazing!</a>
+        </div>
+      </div>
+    </aside>
     <!-- contact me container -->
+    <section id="contact">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-8 offset-lg-2 text-center">
+            <h2 class="section-heading">My Contact Information</h2>
+            <hr class="1">
+            <p>Here's my phone number and email address, because hey that's part of the template and why would you not
+              include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring
+              in
+              from all the companies wanting my insane website design skills.</p>
+            <p>Honestly this template does look really nice, though.</p>
+          </div>
+          <div class="col-lg-6 col-md-6  text-center">
+            <i class="fa fa-phone fa-3x wow bounceIn" style="visibility: visible; animation-name: bounceIn;"></i>
+            <p>+1-608-317-1888</p>
+          </div>
+          <div class="col-lg-6 col-md-6  text-center">
+            <i class="fa fa-envelope-o fa-3x wow bounceIn" data-wow-delay=".1s"
+              style="visibility: visible; animation-delay: 0.1s; animation-name: bounceIn;"></i>
+            <p><a href="mailto:tab@report.ly">holy@shit.com</a></p>
+          </div>
+        </div>
+      </div>
+    </section>
 
-    <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
+  <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous">
+  </script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
+    integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous">
+  </script>
+  <script src="https://code.jquery.com/jquery-3.5.1.js" integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc="
+    crossorigin="anonymous"></script>
 
-    <!-- 
-      All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
 
+  <!-- 
+      All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
+    
       If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
       Your custom JavaScript
      -->
-  </body>
-</html>
+</body>
+
+</html>
```

Comment:
Superb work Shandy you have managed to implement basically every feature that we are asking you to do. 
The only things that I would like you to come back and fix, are the positions of your bottom hr
As well as implementing the 'Google That ***' Button , when you click goto the right link.
Lastly I would like you to implement the on scroll event listener so that your navigation bar changes colours as you scroll a certain distance. 

But well done and keep it up!